src/auth_service/user_management/claims_repository/core/seed.py

```diff
@@ -105,7 +123,44 @@ async def _add_configured_data_steward_claims(
                 continue
             else:
                 log.warning("Added missing data steward with external ID %r.", ext_id)
-        claim = create_data_steward_claim(user.id)
+        else:
+            _check_data_steward_info(data_steward, user)
+        # add the IVA of the data steward
+        iva_data = IvaBasicData(
+            type=data_steward.iva_type, value=data_steward.iva_value
+        )
+        try:
+            iva = await iva_dao.find_one(
+                mapping={
+                    "user_id": user.id,
+                    "type": iva_data.type,
+                    "value": iva_data.value,
+                }
+            )
+        except MultipleHitsFoundError:
+            log.warning(
+                "IVA %s: %r is not unique for user with external ID %r.",
+                iva_data.type,
+                iva_data.value,
+                ext_id,
+            )
+            continue
+        except NoHitsFoundError:
+            try:
+                iva = await _add_iva_for_user(user.id, iva_data, iva_dao=iva_dao)
```

Comment:
Should this also send a notification, as the new IVA is added outside of the API interaction and unverified by default?

Reply:
I think it's not needed. The admin adding the data steward entry in the config would need to communicate with the data steward anyway. And the data steward can see that there is an unverified IVA after logging in.

I do not want to put too much effort into this seeding code, since coupling the seeding with the startup of the claims repo is a very crude solution that we will probably change anyway.